src/collection/views/EditCollection.tsx

```diff
@@ -49,7 +49,7 @@ export const USER_GROUPS: string[] = ['Docent', 'Leering', 'VIAA medewerker', 'U
 
 const EditCollection: FunctionComponent<EditCollectionProps> = props => {
 	const [triggerCollectionUpdate] = useMutation(UPDATE_COLLECTION);
-	const [triggerCollectionDelete] = useMutation(DELETE_COLLECTION);
+	// const [triggerCollectionDelete] = useMutation(DELETE_COLLECTION);
```

Comment:
waarom worden deze in commentaar gezet?

Reply:
Was een lint warning dacht ik

Reply:
ah, ja idd
die delete werkte nog niet omdat er blijkbaar geen `cascade on delete` op stond